docs/schema/actionname-element-in-arrayofactions-layoutdefinitions-schema.md

```diff
@@ -53,7 +54,7 @@ None.
 </thead>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="actionnames-element-in-obpart-layoutdefinitions-schema.md">ActionNames element in OBPart (LayoutDefinitions Schema)</a></span></p></td>
+<td align="left"><a href="actionnames-element-in-obpart-layoutdefinitions-schema.md">ActionNames element in OBPart (LayoutDefinitions Schema)</a></p></td>
```

Comment:
Let's leave the "<p>" in there, unless it creates a validation warning.

Reply:
What was in the " "? I'm thinking the opening <p> tag right?

Reply:
yes, the `<p>` right after /<td align="left"/>